docs/odoo/usage/install-module.rst

```diff
@@ -7,12 +7,35 @@
 
 .. warning:: ``Import Module`` tool (import from a zip  file) doesn't work for modules with python files. It means that it doesn't work in most cases
 
+.. image:: ../../images/import-11.png
+
+.. image:: ../../images/import-11-1.png
+
 From zip archive
 ================
 
 * unzip module to your addons folder
 * restart odoo server
 
+11.0+
+-----
+
+install
+~~~~~~~
+
+* :doc:`activate developer mode <debug-mode>`
+* navigate to ``Apps`` menu
+* click ``Update Apps List`` 
+* search and open a module you need
+* click ``[Install]``
+
+update
+~~~~~~
+
+* :doc:`activate developer mode <debug-mode>`
+* navigate to ``Apps`` menu
+* search and open a module you need
+* click ``Upgrade``
```

Comment:
Fix format https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#buttons

Reply:
Fix in other sections too

Reply:
@yelizariev I don't agree with you. I think I should add <<click Update Apps List>> after  <<navigate to ``Apps`` menu>>

Reply:
My question is about formatting, what are you talking about?